src/crawlee/crawlers/_playwright/_types.py

```diff
@@ -42,6 +44,11 @@ class PlaywrightHttpResponse:
     def read(self) -> bytes:
         return self._content
 
+    async def iter_bytes(self, chunk_size: int | None = None) -> AsyncGenerator[bytes, None]:
```

Comment:
Please add a comment that makes it clear that we don't actually stream anything here, intentionally.